lib/measured/rails/validations.rb

```diff
@@ -6,13 +14,40 @@ def validate_each(record, attribute, measurable)
     measurable_unit = record.send("#{ attribute }_unit")
     measurable_value = record.send("#{ attribute }_value")
 
-    message = options[:message] || "is not a valid unit"
+    return unless measurable_unit.present? || measurable_value.present?
+
+    record.errors.add(attribute, message) if [measurable_unit.blank?, measurable_value.blank?].any?
```

Comment:
It's not clear. We tried a couple approaches. First one was exclusive or, which is really what we are trying to do here.

If you set `nil cm` or `10 nil` that makes no sense, and we want to error. If it's `nil nil` that's fine then it's blank, as bailed above, and if it's `10 cm` obvs that's fine.

Suggestions welcome on a cleaner syntax here.